src/components/vr-accessibility/vr-accessibility.component.ts

```diff
@@ -32,7 +32,7 @@ export class VRAccessibilityComponent implements OnInit, OnChanges, OnDestroy, A
 
   ngAfterViewInit(){
     const scene = this.theScene.nativeElement;
-    this.vrHapticPlot.init(scene, [0, 1, 4, 9, 16, 25, 36, 49, 64, 81, 100]);
+    this.vrHapticPlot.init(scene, [...Array(27).keys()]);
```

Comment:
I think it will be useful to try to integrate some randomly generated data here to see what it will look like in a more realistic use case.  Maybe in a follow up PR.

Reply:
Agreed! This one is laid out to test each of the audio files.